campaigns.json

```diff
@@ -1296,7 +1337,11 @@
         "max": 0, 
         "pending": 0
       }
-    }, 
+    },
+   "SiteBlacklist": [
+         "T2_CH_CERN",
+         "T2_CH_CERN_HLT"
+               ], 
```

Comment:
Scarlet, can you please fix all the problematic indendations in the PR? It really makes it hard to review it. This JSON 
file uses 2 space an indentation convention, can you please stick to it? e.g.
```
  "SiteBlacklist": [
      "T2_CH_CERN",
      "T2_CH_CERN_HLT"
  ],
```

Reply:
Please tell me how to get this because when I hit enter it always goes way over